core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -153,19 +208,234 @@ describe('Learner dashboard functionality', function() {
     await libraryPage.findExploration(EXPLORATION_FRACTION);
     await libraryPage.addSelectedExplorationToPlaylist();
     await learnerDashboardPage.get();
-    await learnerDashboardPage.navigateToPlayLaterExplorationSection();
+    await learnerDashboardPage.navigateToCommunityLessonsSection();
+    await driver.findElement(
+      by.css('.protractor-test-play-later-section'));
     await learnerDashboardPage.expectTitleOfExplorationSummaryTileToMatch(
       EXPLORATION_FRACTION);
     await libraryPage.get();
     await libraryPage.findExploration(EXPLORATION_SINGING);
     await libraryPage.addSelectedExplorationToPlaylist();
     await learnerDashboardPage.get();
-    await learnerDashboardPage.navigateToPlayLaterExplorationSection();
+    await learnerDashboardPage.navigateToCommunityLessonsSection();
+    await driver.findElement(
+      by.css('.protractor-test-play-later-section'));
     await learnerDashboardPage.expectTitleOfExplorationSummaryTileToMatch(
       EXPLORATION_SINGING);
     await users.logout();
   });
 
+  it('should display all the topics on server in edit goals and suggested ' +
+    'for you section and selected topics in current goals and continue where ' +
```

Comment:
All the topics won't be displayed in the 'Suggested for you' section, right?
If name gets too long, just rename to 'displays correct topics in ....' and add comments in the test on what they should be .

Reply:
Done.